### ISSUE TITLE
Syntax error in QueryFactory.php

### ISSUE THREAD

**Issue by kalinchernev** [pfrenssen/matomo-reporting-api#1]
While trying to install joinup dev locally, I got the following error:

```
PHP Parse error:  syntax error, unexpected 'unset' (T_UNSET), expecting identifier (T_STRING) in /var/www/joinup.dev/vendor/pfrenssen/piwik-reporting-api/src/QueryFactory.php on line 80
```

PHP version:
```
PHP 5.6.31-1~ubuntu16.04.1+deb.sury.org+1 (cli) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
    with Xdebug v2.5.5, Copyright (c) 2002-2017, by Derick Rethans
```

**Comment by pfrenssen** [comment]
Thanks for the report! This has been fixed in #2.

**Comment by kalinchernev** [comment]
@pfrenssen please don't forget to publish changes :)
Thanks!
